image/manifest.go

```diff
@@ -44,6 +44,8 @@ type genericManifest interface {
 	// UpdatedImage returns a types.Image modified according to options.
 	// This does not change the state of the original Image object.
 	UpdatedImage(ctx context.Context, options types.ManifestUpdateOptions) (types.Image, error)
+	// SupportsEncryption returns if encryption is supported for the manifest type
+	SupportsEncryption(ctx context.Context) bool
```

Comment:
See the 
> Well… what actually matters is whether encryption is supported by the manifest type that ends up being written to the destination 

comment elsewhere, I’m not sure this is a good long-term API — this way one can only ask about support for an existing image _soruce_, not for a hypothetical _destination_.